### PR TITLE
fixing README + spelling + false links

### PR DESCRIPTION
**Pull request by jackson-07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,18 @@ The Inventory Management System is designed to streamline and optimize the proce
 
 This application will be most useful in smaller to medium-sized businesses wanting to better improve the management of their inventory/stock levels. It will be a tool that keeps everything accurate and up-to-date.
 
-## Screenshot(s)
+## Screenshot
 
-![placeholder-image](path-to-image)
+![placeholder-image](/main_app/images/screenshot.png)
```
Comment (kiranrsp) [inline README.md:9]: Readme looks good

**Comment by jackson-07** [comment]
Hands in everyone!

File: README.md (hunk 1 above)
Comment (MartyEternal) [inline README.md:9]: Finally have a screenshot!

**Comment by MartyEternal** [comment]
> Hands in everyone!

HANDS IN!!

File: main_app/templates/about.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
         <div class="p-5 mb-4 bg-body-tertiary rounded-3">
           <div class="container-fluid py-1">
             <h2>Invinity - Inventory Software Made Easy</h2>
```
Comment (kiranrsp) [inline main_app/templates/about.html:15]: About  page text looks a lot better now

**Review by Exiled184 — APPROVED** [review]
Well done team, great effort all around.

**Pull request merged**